2주차/10989.py

```diff
@@ -0,0 +1,13 @@
+import sys
+
+n=int(sys.stdin.readline())
+cnt = [0]*10001
+
+for i in range(n):
+    x=int(sys.stdin.readline())
+    cnt[x]+=1
+
+for i in range(len(cnt)):
+    if cnt[i]!=0:
+        for _ in range(cnt[i]):
+            print(i)
```

Comment:
```suggestion
for index, num in enumerate(cnt):
    if num != 0:
        for _ in range(num):
            print(index)
```

이런식으로 enumerate도 써보세요~